devices/stm32f0x2.yaml

```diff
@@ -65,6 +65,7 @@ _include:
  - ../peripherals/pwr/pwr_v1.yaml
  - ../peripherals/iwdg/iwdg_with_WINR.yaml
  - ../peripherals/exti/exti.yaml
+ - common_patches/tim/tim14_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0091, TIM14 does not have OPM field in CR1.

Reply:
Fixed